config/database.php

```diff
@@ -1,108 +0,0 @@
-<?php
```

Comment:
we should discard this changes as we shouldn't be changing the default database.php file